README.rst

```diff
@@ -3,5 +3,15 @@ ZeroZero
 
 A documentation repos to describe the common philosophy behind the educational *Zero projects.
 
-
 Not a lot here, but this will change soon enough.
+
+*Zero Projects
```

Comment:
It was - (see usage in the PR title), but I see that RST parsed it as a link (!?)